### PR TITLE
fix(grafana): adds missing panel for tx fetcher

### PR DESCRIPTION
**Pull request by emhane**
Add panel for monitoring duration of fetching pending hashes algorithm.

### CODE REVIEW

File: etc/grafana/dashboards/reth-mempool.json [hunk 1]
```diff
@@ -2324,7 +2324,7 @@
         "type": "prometheus",
         "uid": "${DS_PROMETHEUS}"
       },
-      "description": "Frequency of transaction types seen in announcements",
+      "description": "Duration of one call to fetch hashes pending fetch.\n\nFind Peer - find an idle fallback peer for a hash pending fetch.\n\nFill Request - fill `GetPooledTransactions` request, for the found peer, with more hashes from cache of hashes pending fetch. ",
```
Comment (onbjerg) [inline etc/grafana/dashboards/reth-mempool.json:2327]: i'm not sure I follow what this means:

"Duration of one call to fetch hashes pending fetch"

can you rephrase?

File: etc/grafana/dashboards/reth-mempool.json [hunk 2]
```diff
@@ -2425,31 +2425,117 @@
...
+      ],
+      "title": "Fetch Hashes Pending Fetch Duration",
+      "type": "timeseries"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${DS_PROMETHEUS}"
+      },
+      "description": "Durations of one call to poll `NetworkManager` future, and its nested function calls.\n\nNetwork Handle Message - stream network handle messages from `TransactionsManager`;\nSwarm Events - stream transaction gossip from `Swarm`",
```
Comment (onbjerg) [inline etc/grafana/dashboards/reth-mempool.json:2447]: since this is user facing i'd err on not naming structs etc, but not a blocker

File: etc/grafana/dashboards/reth-mempool.json [hunk 3]
```diff
@@ -2459,26 +2545,43 @@
...
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "disableTextWrap": false,
+          "editorMode": "builder",
+          "expr": "reth_network_duration_poll_network_manager{instance=\"$instance\"}",
+          "fullMetaSearch": false,
+          "hide": false,
+          "includeNullMetadata": true,
+          "instant": false,
+          "legendFormat": "Total Network Manager Future",
```
Comment (onbjerg) [inline etc/grafana/dashboards/reth-mempool.json:2570]: same here

File: etc/grafana/dashboards/reth-mempool.json (hunk 2 above)
Comment (emhane) [reply etc/grafana/dashboards/reth-mempool.json:2447]: yeah in general same...don't think these types will disappear though, hence went for it. also see major benefits of lowering barriers to entry for contributors looking at public dashboard, hence don't have the long metric names to search the codebase to match the panel with the code.

**Review by onbjerg — APPROVED** [review]
